airflow/contrib/kubernetes/secret.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class Secret:
+    """Defines Kubernetes Secret Containers"""
```

Comment:
Is this referring to a Secret Volume Mount?

Reply:
@foxish Yes, this allows users to mount existing secrets to any pod they are deploying.

Reply:
I think the comment/doc string should be changed in that case - to "Defines Kubernetes Secret Volumes" since it's not a container by itself.